### DISCUSSION TITLE
unbiased template and when to preprocess dataset?

### DISCUSSION THREAD

**Discussion by geowk** [CoBrALab/RABIES#291] [General]
Hello,

This question is about registration to the atlas template (Sigma Template for a group of subjects (rats).

We have a dataset of 18 rodents. Three scanned on 6 different days. Is it better to preprocess all of the 18 rodents at the same time using the unbiased template process (fast_commonspace=false option) to ensure less variability in terms of registration from subject to subject?

Using the unbiased template procedure, would it ensure the most optimal registration across subject to subject to an atlas template preprocessing all of the data at once?

thanks.

g.

**Comment by Gab-D-G** [comment]
Hi,

In terms of obtaining highest quality of registration, we recommend the unbiased template method (although it can lack some flexibility and take more time to run than fast_commonspace=true). 

The main case I can think of where you should not try to register the entire dataset at once is if you have very different brain morphology between scans. For instance, if some of your scans were in pups, and others in adults, you should not register them together, and use a proper reference atlas for the pup brains separately.

Otherwise, the unbiased template should reduce cross-scan variability in alignment, since it leverages the brain contrast of the imaging protocol in the dataset for alignment instead of the reference atlas, which has likely a different image contrast.

Best,
Gabriel DG

**Comment by geowk** [comment]
Thanks, Gabriel.
